.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+_Before you submit a pull request, please make sure you have to following:_
+
+- [ ] The title of this pull request offers a good description of what is changed (as it is used in release notes).
+- [ ] Your branch follows the [GitFlow](https://datasift.github.io/gitflow/IntroducingGitFlow.html) branching model.
+- [ ] The code follows the [best practices](#).
+- [ ] The functionality has been manually tested (if applicable)
```

Comment:
I would add an extra line that the install and the update workflow should be tested as well.